spacewalk/package/spacewalk.spec

```
@@ -70,7 +70,7 @@ Requires:       spacewalk-backend-xmlrpc
 Requires:       spacewalk-certs-tools
 
 # Misc
-%if !0%{?rhel}
+%if !0%{?opensuse}
```

```suggestion
%if 0%{?opensuse}
```
This is the right thing, if we only want this for opensuse